crates/ra_ide/src/hover.rs

```diff
@@ -53,10 +103,20 @@ impl HoverResult {
         &self.results
     }
 
+    pub fn actions(&self) -> &[HoverAction] {
+        &self.actions
+    }
+
+    pub fn append_action(&mut self, action: HoverAction) {
+        self.actions.push(action);
+    }
+
     /// Returns the results converted into markup
     /// for displaying in a UI
+    ///
+    /// Does not process actions!
     pub fn to_markup(&self) -> String {
-        self.results.join("\n\n---\n")
+        self.results.join("\n\n___\n")
```

Comment:
Weren't we changing this back & forth a couple of PRs ago?

Reply:
As far as I can see there are some places where '---' were replaced by '___' but not this one.